package.json

```diff
@@ -11,7 +11,7 @@
     "clean": "yarn workspaces foreach -pi run clean",
     "start": "ts-node -T --dir scripts start_studio",
     "setup": "ts-node -T --dir scripts create_default_env --location ./.env.debug",
-    "package": "yarn clean && yarn build && ts-node -T --dir scripts package",
+    "package": "ts-node -T --dir scripts package",
```

Comment:
Why remove the clean and build part here? It makes more sense to remove the old dist and then re-build the project to be always sure the resulting bundle only contains the right files.

Reply:
Because when I type the command "package", I expect it to just package what is there. If we need to clean and build beforehand, then we can type those commands in succession. Typing a command shouldn't be a "surprise". When you build, it builds, when you clean, it cleans. 

We added gulp in the main repository to handle special commands like `yarn cmd build:core` so we can add commands without denaturing the "official" commands (when you yarn build, it builds everything, and we keep the scripts list small)